src/plugins/intel_gpu/tests/unit/test_cases/eltwise_gpu_test.cpp

```diff
@@ -3176,6 +3176,57 @@ TEST(eltwise_gpu_f32, broadcast_test_in4x4x2x2) {
     }
 }
 
+
+TEST(eltwise_gpu_f32, broadcast_test_dim3_dim4) {
+    auto& engine = get_test_engine();
+
+    ov::Shape in_shape = {2, 4, 2};
+    ov::Shape in2_shape = {1, 1, 4, 1};
+    auto input = engine.allocate_memory({ ov::PartialShape(in_shape), data_types::f32, format::bfyx });
+    auto input2 = engine.allocate_memory({ ov::PartialShape(in2_shape), data_types::f32, format::bfyx });
+
+    topology topology;
+    topology.add(input_layout("input", input->get_layout()));
+    topology.add(input_layout("input2", input2->get_layout()));
+    topology.add(eltwise("eltwise", { input_info("input"), input_info("input2") }, eltwise_mode::sum));
+
+    set_values(input, {
+        1.f,   0.f, 5.f, 1.5f,
+        2.f,   0.f, 6.f, 5.2f,
+        3.f,  0.5f, 7.f, 12.f,
+        4.f, -0.5f, 8.f,  8.f
+    });
+
+    set_values(input2, {
+        0.5f,   2.5f,  0.5f,  2.5f
+    });
+
+    network network(engine, topology, get_test_default_config(engine));
+
+    network.set_input_data("input", input);
+    network.set_input_data("input2", input2);
+    auto outputs = network.execute();
+
+    ASSERT_EQ(outputs.size(), size_t(1));
+    ASSERT_EQ(outputs.begin()->first, "eltwise");
+
+    auto output = outputs.at("eltwise").get_memory();
+
+    float answers[16] = {
+        1.5, 7.5, 2.5, 8.5,
+        3.5, 9.5, 4.5, 10.5,
+        1.5, 7.5, 2.5, 8.5,
+        3.5, 9.5, 4.5, 10.5
```

Comment:
could you explain how this answer is created from the two inputs above? it is difficult to understand why sum of those two tensors are like this. For example, max(input) = 12.0. min(input2) = 0.5. But why max(input + input2) is only 10.5?

Reply:
Fixed

Reply:
I found that the main difference is `allow_new_shape_infer`. Does this mean that this test will fail with "old shape infer"? Then, shouldn't we fix that too?